libraries/lib-channel/Channel.cpp

```diff
@@ -64,3 +67,25 @@ size_t Channel::GetChannelIndex() const
 }
 
 ChannelGroup::~ChannelGroup() = default;
+
+double ChannelGroup::GetStartTime() const
+{
+   auto range = Intervals();
+   if (range.empty())
+      return 0;
+   return std::accumulate(range.first, range.second,
+      std::numeric_limits<double>::max(),
+      [](double acc, const auto &pInterval){
+         return std::min(acc, pInterval->Start()); });
+}
+
+double ChannelGroup::GetEndTime() const
+{
+   auto range = Intervals();
+   if (range.empty())
+      return 0;
+   return std::accumulate(range.first, range.second,
+      std::numeric_limits<double>::min(),
```

Comment:
Should it be simply `.0`? Or if time is allowed to be negative should it be `std::numeric_limits<double>::lowest()`?
> For floating-point types with denormalization, min() returns the minimum positive normalized value

Reply:
Thanks!  You are right.  I remembered this in other commits of this pull request but forgot it here.